src/pages/index.hbs

```diff
@@ -7,9 +7,11 @@ order: 0
 <div class="mzp-c-article">
   <div class="mzp-c-article-intro">
     <p>Protocol is a design system for Mozilla branded websites. It establishes a common design language, provides reusable coded components, and outlines high level guidelines for content and accessibility.</p>
+
+    <p>Protocol is still in an <strong>early stage of development</strong> and likely to change a lot in the near future. If you're interested in using it on your project, let us know and we can help you.</p>
```

Comment:
I thought about mentioning Slack here, but that's not open to the public so I wanted some other means of contact as well. We could set up an official mailing list but I doubt it would see much activity. So I think we just need an email alias that sends to the core team, which we should probably set up anyway (and also figure out who the "core team" is). I'll work on that.

Reply:
If we did want to mention Slack we could also make clear it's for NDA Mozillians and staff only. A GitHub issue template for asking questions may also be a good public channel?

Reply:
I still want to set up an email alias but until then I've mentioned #design-systems on Slack and added a link to github issues.